.travis.yml

```diff
@@ -4,6 +4,7 @@ services:
 language: bash
 script:
   - make shellcheck
+  - make get-cluster-lab-images
```

Comment:
I still wonder why this extra script call is needed. Should be done within the Makefile dependencies.
I don't know the exact order.
The community should be able to run `make sd-image` and the missing depencies should be made first. In this case downloading the cluster lab images.

I'll try to add the makefile dependency instead and trigger a Travis build with it.